config/config-sil-ocpp.yaml

```diff
@@ -161,9 +161,13 @@ active_modules:
       display_message:
         - module_id: display_message
           implementation_id: display_message
-      connector_zero_sink:
+      evse_manager_energy_sink:
         - module_id: grid_connection_point
           implementation_id: external_limits
+        - module_id: evse_manager_1_sink
+          implementation_id: external_limits
```

Comment:
This kind of solution, i.e. to plugin one more instance of an EnergyNode mode in between the energy flow graph is pretty overkill!  It is a complete running process, which won't do more than just receive a command and forward it as some limits.  Maybe we should consider to have a single module for the energy flow which has a much more flexible and expressive configuration.